### PR TITLE
Add reset css, banner component

### PR DESCRIPTION
**Pull request by elayabharath**

### CODE REVIEW

File: lib/components/banner/banner.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  flex: 0 0 auto;
+  width: 100%;
+  height: var(--banner-height);
+  display: flex;
+  align-items: stretch;
+  flex-direction: row;
+  justify-content: center;
+
+  text-align: center;
+  color: #e0e0e0;
```
Comment (marianamarasoiu) [inline lib/components/banner/banner.css:13]: maybe can we just make this white and also turn it into a variable?

File: lib/components/banner/banner.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+#banner.hidden {
+  height: 0;
+  visibility: hidden;
+}
+
+#banner .contents {
+  line-height: var(--banner-height);
+  height: var(--banner-height);
```
Comment (marianamarasoiu) [inline lib/components/banner/banner.css:28]: this makes me so happy, thank you!

**Review by marianamarasoiu — APPROVED** [review]
LGTM! thanks @elayabharath !

File: lib/components/banner/banner.css (hunk 1 above)
Comment (elayabharath) [reply lib/components/banner/banner.css]: Ah missed this, fixed!

**Pull request merged**